com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -57,6 +56,7 @@ private enum State
         private NetworkDriver m_Driver;
         private List<INetworkParameter> m_NetworkParameters;
         private byte[] m_MessageBuffer;
+        private NetworkConnection m_ServerConnection;
```

Comment:
Why is this change part of the PR ?

Reply:
we use it in the `ProcessEvent()` method so had to make it into a member variable.